projects/material-css-vars/src/lib/_variables.scss

```diff
@@ -4,222 +4,333 @@
-          --dark-dividers: $dark-dividers,
-          --dark-focused: $dark-focused,
-          --light-primary-text: $light-primary-text,
-          --light-secondary-text: $light-secondary-text,
-          --light-accent-text: $light-primary-text,
-          --light-warn-text: $light-primary-text,
-          --light-disabled-text: $light-disabled-text,
-          --light-dividers: $light-dividers,
-          --light-focused: $light-focused) !default;
+        ( --dark-primary-text-no-rgb: $dark-primary-text-no-rgb,
+          --dark-secondary-text-no-rgb: $dark-secondary-text-no-rgb,
+          --dark-accent-text-no-rgb: $dark-primary-text-no-rgb,
+          --dark-warn-text-no-rgb: $dark-primary-text-no-rgb,
+          --dark-disabled-text-no-rgb: $dark-disabled-text-no-rgb,
+          --dark-dividers-no-rgb: $dark-dividers-no-rgb,
+          --dark-focused-no-rgb: $dark-focused-no-rgb,
+          --light-primary-text-no-rgb: $light-primary-text-no-rgb,
+          --light-secondary-text-no-rgb: $light-secondary-text-no-rgb,
+          --light-accent-text-no-rgb: $light-primary-text-no-rgb,
+          --light-warn-text-no-rgb: $light-primary-text-no-rgb,
+          --light-disabled-text-no-rgb: $light-disabled-text-no-rgb,
+          --light-dividers-no-rgb: $light-dividers-no-rgb,
+          --light-focused-no-rgb: $light-focused-no-rgb,
+          --dark-primary-text: rgba(var(--dark-primary-text-no-rgb)),
+          --dark-secondary-text: rgba(var(--dark-secondary-text-no-rgb)),
+          --dark-accent-text: rgba(var(--dark-accent-text-no-rgb)),
+          --dark-warn-text: rgba(var(--dark-warn-text-no-rgb)),
+          --dark-disabled-text: rgba(var(--dark-disabled-text-no-rgb)),
+          --dark-dividers: rgba(var(--dark-dividers-no-rgb)),
+          --dark-focused: rgba(var(--dark-focused-no-rgb)),
+          --light-primary-text: rgba(var(--light-primary-text-no-rgb)),
+          --light-secondary-text: rgba(var(--light-secondary-text-no-rgb)),
+          --light-accent-text: rgba(var(--light-accent-text-no-rgb)),
+          --light-warn-text: rgba(var(--light-warn-text-no-rgb)),
+          --light-disabled-text: rgba(var(--light-disabled-text-no-rgb)),
+          --light-dividers: rgba(var(--light-dividers-no-rgb)),
+          --light-focused: rgba(var(--light-focused-no-rgb))
+          ) !default;
 
 $default-light-theme:
   (
     // PRIMARY
-    --palette-primary-50: public-color-util.hex-to-rgb(#e1f5fe),
-    --palette-primary-100: public-color-util.hex-to-rgb(#b3e5fc),
+
+    --palette-primary-50-no-rgb: public-color-util.hex-to-rgb(#e1f5fe),
+    --palette-primary-100-no-rgb: public-color-util.hex-to-rgb(#b3e5fc),
+    //lighter
+    --palette-primary-200-no-rgb: public-color-util.hex-to-rgb(#81d4fa),
+    --palette-primary-300-no-rgb: public-color-util.hex-to-rgb(#4fc3f7),
+    --palette-primary-400-no-rgb: public-color-util.hex-to-rgb(#29b6f6),
+    --palette-primary--no-rgb: public-color-util.hex-to-rgb(#03a9f4),
```

Comment:
There is a typo here in the variable name, it needs to be `--palette-primary-500-no-rgb`